localConfig_template.json

```diff
@@ -0,0 +1,29 @@
+{
+    "port": 3000,
+    "listeningInterface": "127.0.0.1",
+    "scheduleUpdate": {
+
+        "enabled": {
+            "@detail": {
+                "input_type": "option",
+                "options": {
+                    "true": true,
+                    "false": false
+                },
+                "skip_next_if": false
```

Comment:
When configuring with scheduler disabled, if later this is changed to true, the time is undefined

I'm guessing this line is the issue.

Reply:
Yes it is true ! It was made for a one time configuration, maybe I'll add a new feature to the module for reconfiguring.
I'll delete the line to force the scheduler configuration anyway